falcon/request_helpers.py

```diff
@@ -15,9 +15,9 @@
 """Utilities for the Request class."""
 
 from http import cookies as http_cookies
-import io
 import re
 
+from falcon.stream import BoundedStream
```

Comment:
It would be nice to have TODO comment here to remind us it is here for backwards-compatibility and it should be removed in Falcon 4.0

Reply:
Okay.